clients/python/llmengine/data_types/rest.py

```diff
@@ -164,6 +165,7 @@ class CreateLLMEndpointRequest(BaseModel):
     memory: Optional[StorageSpecificationType]
     gpu_type: Optional[GpuType]
     storage: Optional[StorageSpecificationType]
+    nodes_per_worker: Optional[int] = None
```

Comment:
is this something we want users to be able to provide?

Reply:
yes, I view it in the same vein as gpus/gpu_type/the other resource requests

Reply:
we do have a route to figure out this number for users if they don't set any resource requests (through the "default resource requests" behavior), e.g. someone creating a 405b unquantized without specifying any resource requests will default to multinode. Otherwise I think the flexibility/explicitness is probably good?